### PR TITLE
Docs: Update ReadMe

### PR DESCRIPTION
**Pull request by Husain01**
- Updated ReadMe
- Link: https://github.com/Husain01/Kitabook/tree/Add-Readme#readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,63 @@
...
+</h1>
+
+
+# **📚 KITABOOK**
+### KITABOOK | Helping new readers hook up with books
+<br/>
+
+<!-- PROJECT SHIELDS -->
+<p align="center">
+  <!-- <a href="https://github.com/Husain01/Neu-UI/blob/main/LICENSE" target="blank">
```
Comment (nshanbhag20001412) [inline README.md:15]: you can remove the unwanted code from here

**Review by nshanbhag20001412 — APPROVED** [review]
rest looks good to me

File: README.md (hunk 1 above)
Comment (Husain01) [reply README.md:15]: Will be adding it later that's why have commented.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,63 @@
-# KITABOOK
- KITABOOK | Helping new readers hook up with books
+<h1 align="center">
+<br />
+<a href="https://vispaui-docs-development.netlify.app/"><img src="https://raw.githubusercontent.com/Husain01/Kitabook/Dev/assets/KITABOOK.png" alt="Neu UI" width="300"></a>
```
Comment (RakshaPawar108) [inline README.md]: Here, I guess you are putting the wrong link. Correct me if I am wrong.

**Review by Sudhanshusoni45 — APPROVED** [review]

**Review by RakshaPawar108 — APPROVED** [review]
Just a small fix suggested. Other than that, the Readme file looks good. Great work!

File: README.md (hunk 2 above)
Comment (Husain01) [reply README.md]: Oops. Thank you so much

**Pull request merged**